styleguide/source/_patterns/02-molecules/upcoming-event.twig

```diff
@@ -0,0 +1,20 @@
+<div class="ama__upcoming-event">
+  {% if event.link %}
+    {% set link = event.link|merge({class: 'ama__link--homepage', target: 'self'}) %}
+    <h5 class="ama__h5 ama__h5--homepage ama__upcoming-event__title">
+      {% include "@atoms/link/link.twig" %}
```

Comment:
Could we just extend the h5 here instead of wrapping it in an h5 tag?

Reply:
For page hierarchy this should be a heading.